draft-ietf-rats-architecture.md

```diff
@@ -327,7 +327,38 @@ implicitly trusted is often referred to as a Root of Trust.
 
 # Freshness
 
-    <this section can include some high-level content from draft-birkholz-rats-reference-interaction-model>
+It is important to prevent replay attacks where an attacker replays
+old Evidence or an old Attestation Result that is no longer correct.
+To do so, some mechanism of ensuring that the Evidence and Attestation
+Result are fresh, meaning that there is some degree of assurance
+that they still reflect the latest state of the Attester, and that any Attestation
+Result was generated using the latest Appraisal Policy for Evidence.
+There is, however, always a race condition possible in that the
+state of the Attester, and the Appraisal Policy for Evidence, may
+change immediately after the Evidence or Attestation Result was generated.
+The goal is merely to narrow the time window to something the Verifier
+(for Evidence) or Relying Party (for an Attestation Result) is willing to accept.
+
+There are two common approaches to providing some assurance of freshness.
+The first approach is that a nonce is generated by a remote entity
+(e.g., the Verifier for Evidence, or the Relying Party for an Attestation
+Result), and the nonce is then signed and included along with the claims in
+the Evidence or Attestation Result, so that the remote entity knows that the
+claims were signed after the nonce was generated.
+
+A second approach is to rely on synchronized clocks, and include a signed
+timestamp (e.g., using {{?I.D.draft-birkholz-rats-tuda}}) along with the
+claims in the Evidence or Attestation Result, so that the remote entity knows
+that the claims were signed at that time, as long as it has some assurance that
+the timestamp is correct.  This typically requires additional claims about
+the signer's time synchronization mechanism in order to provide such assurance.
+
+In either approach, it is important to note that the actual values in claims
+might have been generated long before the claims are signed.  If so, it is the
+signer's responsibility to ensure that the values are still correct when they
+are signed.  For example, values might have been generated at boot, and then
+used in claims as long as the signer can guarantee that they cannot have changed
+since boot.
```

Comment:
Boot-Cycle is a good example I think (but only before the entropy of user-space kicks in. There are other problems regarding the "stability" of these "early boot measurements" today, but that is beside the point here, I think). But still leaves the question open how such a guarantee looks like.

In summary there seem to be four relevant points on time here:

1.) creation of the value in the attestedEE
2.) measurement of the value in the attestingEE to create the Claim value
3.) creation of the evidence
    * with creation of the nonce _before_ the creation of the evidence (although I am still unsure why there is an emphasis on that), or
    * with retrieval of a trustworthy timestamp as close as possible before the creation of the evidence
4.) conveyance/retrieval of the evidence to/by an evidence sink (in general the Verifier or a relaying entity)

Which of these (at least) four points in time are now actually vital wrt to expressing the freshness of the thing conveyed? Maybe this should go into the 2nd item of this section after "an elaboration of the concept of freshness"?